rotkehlchen/chain/ethereum/modules/aave/aave.py

```diff
@@ -174,6 +204,39 @@ def _get_balances(
 
             aave_balances[account] = AaveBalances(lending=lending_map, borrowing=borrowing_map)  # type: ignore
 
+        for account, balance_sheet in (  # for v3
+            given_eth_balances if isinstance(given_eth_balances, dict) else given_eth_balances()
+        ).items():
+            for asset, balance in balance_sheet.assets.items():
+                if (
+                    asset.is_evm_token() and
+                    (asset := asset.resolve_to_evm_token()).protocol == CPT_AAVE_V3 and
+                    (reserve_data := self._get_v3_reserve_data(asset)) is not None
+                ):
+                    aave_balances[account].lending[CryptoAsset(
+                        strethaddress_to_identifier(asset.underlying_tokens[0].address),  # type: ignore[attr-defined]  # asset is resolved to evm token
+                    )] = AaveLendingBalance(
```

Comment:
can you check if the underlying tokens is not empty before? I'm not sure if we set it to None or empty list but I see a user modifying it and creating an error here

Reply:
I just noticed that you make this check in `_get_v3_reserve_data`. Add a comment mentioning that is checked there